### DISCUSSION TITLE
Please help me code my way into getting this addon running! (What's a pip?)

### DISCUSSION THREAD

**Discussion by ilaision** [carson-katri/dream-textures#60] [Q&A] [answered]
So this is what I get when I try installing dependencies without admin access:
PermissionError: [Errno 13] Permission denied: 'C:\\Program Files\\Blender Foundation\\Blender 3.3\\3.3\\python\\Include\\pystate.h'

And this is what I get when I try to install dependencies with admin access:
ERROR: File "setup.py" or "setup.cfg" not found. Directory cannot be installed in editable mode: C:\Users\illie\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream-textures-main\stable_diffusion
WARNING: You are using pip version 21.2.4; however, version 22.2.2 is available.
You should consider upgrading via the 'C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\bin\python.EXE -m pip install --upgrade pip' command.

I went over all the troubleshooting guides, I completely reset my blender fetch, I reinstalled everything, I did everything but the problem is my lack of python experience and here I am coming for help after trying to bypass this Dependencies step for over 2 days. 
I really want to get this addon running but unfortunately I'm a 3D modeler not a python programmer. I don't know what a "pip" is (I tried doing exactly what this warning regarding to my outdated pip said, but the console replied with "invalid syntrax" after every command.) So yeah, I don't know what any of this means but regardless If anyone's willing to help I will greatly appreciate it and will try my best to comply. 

GPU: GTX 1080 TI
CPU I7 9700K 
16 RAM DDR4

**Comment by Umoy** [comment, marked as answer]
Hi :)
The problem isn't the pip itself. 
You just dowloaded the main part of the addon, not the addon itself. You have to download the "dream texture" latest version (also written in the installation guide) 
 https://github.com/carson-katri/dream-textures/releases/tag/0.0.5

but if you want to hide this pip warning.. just type "python.EXE -m pip install --upgrade pip" within the folder: C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\bin\

Had the same issue. I hope this helps.

greetz
Umoy

**Reply by ilaision** [reply]
Hi, Umoy. Thanks for the reply! Unfortunately, I did already do that. I read the installation guide and the troubleshooting guide and installed the correct zip file but the problem persisted exactly the same so I eventually posted this thread..

However, I had to try again now just to make sure everything is already done exactly as explained, and something different happened.
As usual, trying to install the dependencies without admin gives the same outcome:
PermissionError: [Errno 13] Permission denied: 'C:\\Program Files\\Blender Foundation\\Blender 3.3\\3.3\\python\\Include\\pystate.h'

yet with admin, it actually progressed the most so far! But, obviously something else led to its doom:
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    opencv-python==4.6.0.66 from https://files.pythonhosted.org/packages/cf/09/b24c266cd61ddeed101b90c92a26f54d060b06f4a1b102eb891576d6e9e2/opencv_python-4.6.0.66-cp36-abi3-win_amd64.whl#sha256=0dc82a3d8630c099d2f3ac1b1aabee164e8188db54a786abb7a4e27eba309440 (from -r C:\Users\illie\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream_textures\requirements-win-torch-1-11-0.txt (line 10)):
        Expected sha256 0dc82a3d8630c099d2f3ac1b1aabee164e8188db54a786abb7a4e27eba309440
             Got        e8839ad3383155d8f649e55847a2fe6cc8557aa1ee3068bd58704e7f30b0cbb9


Getting closer, but not quite there yet. I don't really know what's wrong with line 10 neither do I know how to update the hashes.
